deploy/crds/operator.ibm.com_authentications_crd.yaml

```diff
@@ -240,6 +248,7 @@ spec:
               required:
               - imageName
               - imageRegistry
+              - bootstrapUserId
```

Comment:
Is `boostrapUserId` a required field or a config field? We can either remove it from this required field list or, if it is required, we can add it to the CR and place it under the imageName, imageRegistry, etc. fields.

Reply:
i have removed it from required list